tests/testthat/test-ui-gantt.R

```diff
@@ -7,20 +7,25 @@ test_that("cXgantt1", {
 
 test_that("cXgantt2", {
     check_ui_test(cXgantt2())
+    warning("background coloring for dependencies field in table is red when null, blue when not null, and in web example, is blue when null and red when not null")
```

Comment:
the table colouring is not a concern, please remove these warnings.

Reply:
removed the warnings for table coloring